eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -73,10 +73,12 @@ case class Commitments(localParams: LocalParams, remoteParams: RemoteParams,
 
   def announceChannel: Boolean = (channelFlags & 0x01) != 0
 
-  def availableBalanceForSendMsat: Long = {
+  def availableBalances: (Long, Long) = {
```

Comment:
It's quite hard to understand what this tuple of `Long` is. I think that `availableBalances` should be typed, with something like:

```scala
case class AvailableBalance(local: MilliSatoshi, remote: MilliSatoshi)
```